apps/oeth/src/App.tsx

```diff
@@ -7,24 +7,20 @@ export const App = () => {
   return (
     <>
       <CssBaseline />
-      <Stack>
+      <Stack minWidth={370}>
         <Topnav />
         <Container
           sx={{
-            mt: {
-              xs: 3,
-              md: 5,
-              paddingInline: {
-                xs: 2,
-                md: 0,
-              },
-            },
+            mt: { xs: 3.25, md: 3 },
+            mb: 10,
+            pt: (theme) => ({
+              xs: '112px',
```

Comment:
nit: seems like this one should be derived on math from the `md` value.
(i don't really care though)

Reply:
this padding accounts for the topnav height which has quite strange dimensions when < md: it wraps to 2 rows, the second one having distinct height than the first. I initially just wanted to do `theme.mixins.toolbar.height * 2` but designs said otherwise 😅